server/routes/aws/cloudwatch_routes.py

```
@@ -0,0 +1,466 @@
+            conn.commit()
+            return updated
+    except Exception:
+        logger.exception("[CLOUDWATCH] Failed to set is_active=%s", active)
+        return False
+
+
+# ---------------------------------------------------------------------------
+# Webhook helpers
+# ---------------------------------------------------------------------------
+
+def _handle_subscription_confirmation(sns_message: dict, user_id: str):
```

Ruff (0.15.14)

[warning] Missing return type annotation for private function `_handle_subscription_confirmation`

Add return type annotation: `None`

(ANN202)